Makefile

```diff
@@ -3,7 +3,7 @@ out/osqtool:
 	GOBIN=$(CURDIR)/out go install github.com/chainguard-dev/osqtool/cmd/osqtool@latest
 
 out/odk-detection.conf: out/osqtool
-	./out/osqtool --max-results=0 --max-total-daily-duration=3h45m --max-query-daily-duration=60m --verify pack detection/ > out/odk-detection.conf
+	./out/osqtool --max-results=0 --max-total-daily-duration=3h45m --max-query-daily-duration=1.5h --verify pack detection/ > out/odk-detection.conf
```

Comment:
I couldn't get this to complete during 'make packs' -- not sure if this is an appropriate change here.